ref/general/deploying-openshift.adoc

```diff
@@ -12,23 +12,22 @@
 :page-type: general
 = Deploying Open Liberty applications on OpenShift
 
-link:https://www.openshift.com/[Red Hat OpenShift] is a Kubernetes-based application platform.
-After you write your applications, you can containerize and deploy them to OpenShift to orchestrate and automate your containers.
-You can also implement the Open Liberty Operator to simplify the process of deploying and managing your applications in the cloud.
+link:https://www.openshift.com/[OpenShift] is a Kubernetes-based application platform.
+After you write an application, you can use OpenShift to build, containerize, and deploy your application so it's highly available and scalable.
+You can use the Open Liberty Operator to simplify the process of deploying and managing your applications in the cloud.
```

Comment:
You can use the Open Liberty Operator to simplify deploying and managing your applications in the cloud.